### DISCUSSION TITLE
arcwelder included?

### DISCUSSION THREAD

**Discussion by Tinchus2009** [supermerill/SuperSlicer#1740] [General]
Hello. I remember seeing some arc welder folder in the source  code project of superslicer. Is arcwelder functionality already included in the slicer?

**Comment by supermerill** [comment]
@lordofhyphens has worked on it. But I don't know much more.

**Reply by Tinchus2009** [reply]
My question is because Im using arcwelder in the output option, but since I saw and "arwelder" folder in the sources, I thought "I might be using something that is already included" ja.

**Comment by tome9111991** [comment]
Hi.
Any news to include arcwelder?

**Reply by supermerill** [reply]
planned but not done.
